### DISCUSSION TITLE
What's the latest stockfish version which supports for iOS 9

### DISCUSSION THREAD

**Discussion by bharath-dev** [official-stockfish/Stockfish#3751] [Q&A]
Currently I am using **SF11** and its working on **iOS 9**, But finding hard to update to SF14.
Is it something to do with iOS base version or XCode 13.0?

Please guide me on it.

**Comment by osteslag** [comment]
What's not working for you? Compilation? Execution? What behaviour/error(s) do you see? Does it work for you on newer iOS versions? Just curious, why do you need to support iOS 9?

SF is C++ and is not doing anything exotic as far as I know. So it should "just work". Note that going from `sf_11` to `sf_14` requires you to support NNUE (and loading a compatible network file at runtime), or somehow work around it by enforcing classical evaluation.

**Comment by nguyenpham** [comment]
My project (open source) supports iOS from 7.0 to current, XCode 13, using both SF 13 and 14:

https://github.com/nguyenpham/banksiagui-ios
